cmd/mailbox-controller/main.go

```diff
@@ -52,6 +52,8 @@ import (
 	"github.com/kcp-dev/logicalcluster/v3"
 
 	clientopts "github.com/kubestellar/kubestellar/pkg/client-options"
+	edgeclusterclientset "github.com/kubestellar/kubestellar/pkg/client/clientset/versioned/cluster"
+	edgev1alpha1informers "github.com/kubestellar/kubestellar/pkg/client/informers/externalversions"
```

Comment:
I would not include "v1alpha1" in this alias because the package path is not that specific.